backend/dataall/db/api/environment.py

```diff
@@ -1285,61 +1299,45 @@ def paginated_environment_redshift_clusters(
         ).to_dict()
 
     @staticmethod
-    def list_environment_objects(session, environment_uri):
-        environment_objects = []
+    def count_environment_objects(session, environment_uri):
```

Comment:
[Clarification] - nice addition, this method is used to check that environment deletion happens after all child objects are deleted. We do not need to list them, just count them